src/utils/validateUtil.ts

```diff
@@ -60,13 +64,16 @@ function convertMessages(
     return target;
   }
   /* eslint-enable */
-
-  return fillTemplate(setValues({}, defaultValidateMessages, messages)) as ValidateMessages;
+  // @NOTE setValues does not seem to be supposed to receive ValidateMessages, bug?
```

Comment:
`defaultValidateMessages` & `messages` is not all the same cause this un-match.

Reply:
It's ok to ignore this since we just reuse `setValues` here.

Reply:
Ok, but imo, it looks pretty confusing when reading the code with the following signature fo setValues:

```ts
setValues<FormValues>(
  store: Store<FormValues>,
  ...restValues: Partial<FormValues>[]
): Store<FormValues> 
```

Might be better to just use another generic function like `assignObject` for generic usage that is not tied to the actual store. This would break when/if the store internal structure changes at some point.

Reply:
`setValues`  is aim to generate a merged store from origin store copy. It will set nest value path by path and not change not affect one.

`fillTemplate` is also the same that we provides default template but developer can change part of them and not affect un-changed part.